src/render.rs

```diff
@@ -91,7 +84,7 @@ impl<'a> Renderer<'a> {
                 // might be a struct or some nested structure
                 if key.contains('.') {
                     let new_key = key.split_terminator('.').skip(1).collect::<Vec<&str>>().join(".");
-                    return value.pointer(&self.get_json_pointer(&new_key)).cloned().ok_or_else(|| FieldNotFound(key.to_string()));
+                    return value.pointer(&get_json_pointer(&new_key)).cloned().ok_or_else(|| FieldNotFound(key.to_string()));
```

Comment:
This line is _really_ long. Either split it up, or use the idiomatic Rust style for cases like this:

```rust
value.pointer(&get_json_pointer(&new_key))
     .cloned()
     .ok_or_else(|| FieldNotFound(key.to_string()));
```